CHANGELOG

```diff
@@ -193,6 +193,9 @@
 2.7.3 (compared to 2.7.2)
```

Comment:
good find it should go to 2.8.1 - i missed the changelog when rebasing from the 2.7 branch